.readthedocs.yaml

```diff
@@ -0,0 +1,27 @@
+# Read the Docs configuration file
```

Comment:
FYI: on my local system, there are complaints about the missing `_static` directory.  I've used a `mkdir -p docs/source/_static` workaround, but maybe it can be removed from the configuration since it's not used.

Reply:
Sure